tests/main/interfaces-log-observe/task.yaml

```diff
@@ -8,11 +8,22 @@ details: |
     list. The plug must not be auto-connected on install and, as usual, must be
     able to be reconnected.
 
+    A snap which defines log-observe (without some additional to-be-defined
+    interface) must not be allowed "control"-type log syslog(2) accesses (i.e.
+    SYSLOG_ACTION{,_READ}_CLEAR, SYSLOG_ACTION_CONSOLE_{ON,OFF,LEVEL})
+
 environment:
     SNAP_NAME: log-observe-consumer
     PLUG: log-observe
 
 prepare: |
+    if os.query is-jammy || os.query is-noble ; then
+        echo "Ensure that on Jammy+, sysctl kernel.dmesg_restrict == 1"
+        # This should be the default, but we write it out just to be doubly sure
+        # On systems before Jammy, leave it as whatever the default is.
+        sysctl -w kernel.dmesg_restrict=1
```

Comment:
Do we need to restore this back to the original value on `restore`?

Reply:
Ah great catch. In practice it *probably* doesn't matter because I don't expect this to ever actually change the setting, but we definitely should just in case. I'll address